packages/sdk/src/solana/types/programs/index.ts

```diff
@@ -64,6 +65,13 @@ export type TokenMetadataInput = z.input<typeof TokenMetadataInputSchema>;
  */
 export type CreatorInput = z.input<typeof CreatorInputSchema>;
 
+/**
+ * @public
+ */
+export type CreatorOutput = Omit<Creator, "address"> & {
+  readonly address: string;
```

Comment:
what's the use of this? does something expect readonly string?

Reply:
The `Creator` type returns `address` as a `PublicKey` - but in our return types it should be a `string` - so the type is being overwritten here.

Reply:
ah i see